src/sections/OurDoctors/TeamSlider.jsx

```diff
@@ -0,0 +1,108 @@
+import Slider from "react-slick";
+import "slick-carousel/slick/slick.css";
+import "slick-carousel/slick/slick-theme.css";
+import image1 from "../../assets/img_01.webp";
+import image2 from "../../assets/img_02.webp";
+import image3 from "../../assets/img_03.webp";
+import image4 from "../../assets/img_04.webp";
+import image5 from "../../assets/img_05.webp";
+import PropTypes from "prop-types";
+import StarRating from "./StarRating";
+
+const teamMembers = [
+  {
+    name: "Jubayer Al Hasan",
+    role: "Marketing Expert",
+    imageUrl: image1,
+    rating: 4,
+  },
+  {
+    name: "Jannatul Ferdaus",
+    role: "Broker",
+    imageUrl: image2,
+    rating: 5,
+  },
+  {
+    name: "Chris Matial",
+    role: "Broker",
+    imageUrl: image3,
+    rating: 3,
+  },
+  {
+    name: "Mark Filo",
+    role: "CEO & Founder",
+    imageUrl: image4,
+    rating: 5,
+  },
+  {
+    name: "Professor Dr D. Filo",
+    role: "CEO & Founder",
+    imageUrl: image5,
+    rating: 4,
+  },
+];
+
+const TeamSlider = () => {
+  const settings = {
+    dots: true,
+    infinite: true,
+    autoplay: true,
+    speed: 500,
+    slidesToShow: 4,
+    slidesToScroll: 3,
+    responsive: [
+      {
+        breakpoint: 1024,
+        settings: {
+          slidesToShow: 3,
+        },
+      },
+      {
+        breakpoint: 768,
+        settings: {
+          slidesToShow: 2,
+        },
+      },
+      {
+        breakpoint: 640,
+        settings: {
+          slidesToShow: 1,
+        },
+      },
+    ],
+  };
+
+  return (
```

Comment:
This slider does not work on mobile screen. In that there is one item visible which is correct but there are only 2 dots, which means that dots/indicators are not responsive.

Reply:
Resolved. thanks!